src/Tribe/Template.php

```diff
@@ -73,6 +73,15 @@ class Tribe__Template {
 	 */
 	protected $template_folder_lookup = false;
 
+	/**
+	 * Create a class variable for the include path, to avoid conflicting with extract.
+	 *
+	 * @since  4.11.0
+	 *
+	 * @var  string
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 84e38e7e5b2646f6903df726a21e01f9</sub>